micromanager_gui/main_window.py

```diff
@@ -270,8 +276,19 @@ def _on_mda_finished(self, sequence: useq.MDASequence):
         self._set_enabled(True)
 
     def browse_cfg(self):
-        self._mmc.unloadAllDevices()  # unload all devicies
-        print(f"Loaded Devices: {self._mmc.getLoadedDevices()}")
+
+        current_cgf = self.cfg_LineEdit.text()
+
+        file_dir = QtW.QFileDialog.getOpenFileName(self, "", "", "cfg(*.cfg)")
+        self.cfg_LineEdit.setText(str(file_dir[0]))
+        self.max_min_val_label.setText("None")
+        self.load_cfg_Button.setEnabled(True)
+
+        if not self.cfg_LineEdit.text():
+            self.cfg_LineEdit.setText(current_cgf)
+            self.load_cfg_Button.setEnabled(False)
```

Comment:
I think this can be simplified like so:
```suggestion
        file_dir = QtW.QFileDialog.getOpenFileName(self, "", "", "cfg(*.cfg)")
        if file_dir:
            self.cfg_LineEdit.setText(str(file_dir[0]))
            self.max_min_val_label.setText("None")
            self.load_cfg_Button.setEnabled(True)
```

eliminating the `else` branch because if nothing is returned there may still be text in the lineedit that we want to be able to load.

Reply:
oops should really be:
```suggestion
        (filename, selectedFilter) = QtW.QFileDialog.getOpenFileName(self, "", "", "cfg(*.cfg)")
        if filename:
            self.cfg_LineEdit.setText(filename)
            self.max_min_val_label.setText("None")
            self.load_cfg_Button.setEnabled(True)
```
in order to not accidently blank out the dialog